src/cordova/android/OutlineAndroidLib/outline/src/main/java/org/outline/vpn/VpnTunnelService.java

```diff
@@ -621,19 +575,9 @@ public int getResourceId(final String name, final String type) {
     return getResources().getIdentifier(name, type, getPackageName());
   }
 
-  /* Returns the server's name from |serverConfig|. If the name is not present, it falls back to the
-   * host name (IP address), or the application name if neither can be retrieved. */
+  /* Returns the server's host name (IP address). */
   private String getServerName(final TunnelConfig config) {
```

Comment:
It looks like this is only used [to build the foreground service notification](https://github.com/Jigsaw-Code/outline-client/blob/3c015bde60d0142b8dc39707aa4ac72f49632ede/src/cordova/android/OutlineAndroidLib/outline/src/main/java/org/outline/vpn/VpnTunnelService.java#L595).

We should be more explicit about that. If we need a name to identify the service we are using, we should pass that down instead of a host address. The name is nice because it's protocol-agnostic. Some strategies may need multiple IP addresses, which doesn't fit the host address model.

It seems we need:
- id (why?)
- service name (for system VPN UI)
- proxy config (for setting up the proxy)